gui/src/components/Taipy/Layout.tsx

```diff
@@ -16,12 +16,29 @@ const baseSx = { display: "grid" };
 
 const HAS_TEXT = /[^\d\s]/;
 const SPLIT_COLS = /\D/;
+const EXPR_COLS = /\D*(\d)+[^\d*]*\*[^\d*]*(\d)+\D*/;
+
+const expandCols = (cols: string) => {
```

Comment:
3foobar1 will not match as we request a * in between

Reply:
@FabienLelaquais Are you happy with this ?

Reply:
Yeah my comment on 3boobar1 was wrong, after I rephrased that ten times...

However: in my opition, "2 1 3*2" should be equivalent to "2 1 2 2 2" which I find will not work, since the match is performed only one. Same for "2*1 3*2" for example (creating "1 1 2 2 2")... Am I missing something?

Reply:
I get your point
Will do

Reply:
I tried to be more clever:
- 1 1 1 1 => 1fr 1fr 1fr 1fr
- 10px 1em 1rem 1fr => 10px 1em 1rem 1fr
- 1*3px 1 3*1em 2 3 3*2 => 3px 1fr 1em 1em 1em 2fr 3fr 2fr 2fr 2fr